### ISSUE TITLE
[ENH] Prefix / Suffix should be changable in ad config

### ISSUE THREAD

**Issue by DreckSoft** [Second-Hand-Friends/kleinanzeigen-bot#121]
In issue #101 a check was introduced to ensure prefix + description + suffix is less than 4000 chars. Now I cannot publish several ads as those exceed the limit. I usually cut the suffix to make it fit. However, it is not possible to overwrite the suffix on a per-ad-basis. Adding suffix to the ad file is simply ignored and the check fails.

**Comment by DreckSoft** [comment]
Hm, that issue was auto-closed. Could someone with the appropriate permission please re-open it? I still think this would be a good enhancement.

**Comment by sebthom** [comment]
Does it actually make sense to specify description prefixes/suffixes at the ad level, or would it be sufficient to simply allow the disabling of the globally specified prefix/suffix per ad?

**Comment by DreckSoft** [comment]
Disabling should be fine. I would then add a shortened footer to the description text. I need to edit that anyway.

**Comment by DreckSoft** [comment]
Setting the suffix in the ad config (empty string or different text) is ignored. I tried that. 
I'm fine with both parsing the suffix in the ad config and overwriting the default or using the disable option.

**Comment by DreckSoft** [comment]
No, I missed that one. I already re-published everything, so the next test has to wait unfortunately. 
But looks good so far from the description, thanks!
